file.c

```diff
@@ -6537,6 +6537,142 @@ const char ruby_null_device[] =
  *  read-only, which is reported as <code>0444</code>.
  *
  *  Various constants for the methods in File can be found in File::Constants.
+ *
+ *  == What's Here
+ *
+ *  First, what's elsewhere. \Class \File:
+ *
+ *  - Inherits from class IO -- in particular, methods for creating,
+ *    reading, and writing files.
+ *  - Includes module FileTest.
+ *
+ *  \Class \File provides methods for:
+ *
+ *  === Creating
+ *
+ *  - ::new:: Creates a new file at the given file path; returns the file.
+ *  - ::open:: Creates a new file at the given file path;
+ *             calls the block with the file.
+ *  - ::mkfifo:: Returns the FIFO file created at the given path.
+ *
+ *  === Deleting
+ *
+ *  - ::delete:: Deletes the file for each given file path.
+ *  - ::truncate:: Truncates the file at the given file path to the given size.
```

Comment:
`::truncate` and `#truncate` potentially modify an existing file.  This is a fundamentally different operation from `::unlink`, which removes a reference to a file from a directory.  I suppose these could go in Settings, which already has `rename`.  Alternatively, maybe just rename the section to `Other`?